media/PartEditor/index.js

```diff
@@ -105,10 +105,10 @@ editor.Editor = class {
       this.updateDocument();
     });
 
-    // set backend by graph-view
+    // show graph view with identical node selection
     this.document.getElementById('circle-graph').addEventListener('click', (e) => {
-      let beName = this.currentBackendName();
-      vscode.postMessage({command: 'selectByGraph', backend: beName});
+      let names = this.getSelectionNames();
+      vscode.postMessage({command: 'selectByGraph', selection: names});
```

Comment:
this is where this PR change is. previous was to let `PartGraphSelector` to toggle backend of the node. With this PR, graph nodes selection will be identical with `Operators` listbox.